WordPress/Sniffs/Files/FileNameSniff.php

```diff
@@ -43,8 +43,12 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 
 		if ( false !== strpos( $fileName, '_' ) ) {
 			$expected = str_replace( '_', '-', $fileName );
-			$error    = 'Filename "' . $fileName . '" with underscores found; use ' . $expected . ' instead';
-			$phpcsFile->addError( $error, $stackPtr, 'UnderscoresNotAllowed' );
+			$error    = 'Filename "%s" with underscores found; use %s instead';
```

Comment:
Could we not directly add the strings to the method instead of first defining a variable?